docs/src/content/docs/de/getting-started.mdx

```diff
@@ -126,4 +126,4 @@ In der wachsenden Liste der Anleitungen in der Seitenleiste findest du Hilfe bei
 
 Wenn du keine Antwort in dieser Dokumentation finden kannst, besuche bitte die [Astro Docs](https://docs.astro.build) für die vollständige Astro-Dokumentation. Deine Frage kann vielleicht beantwortet werden, wenn du verstehst, wie Astro im Allgemeinen funktioniert.
 
-Du kannst auch im [Astro Discord](https://astro.build/chat/) Hilfe von unserer aktiven, freundlichen Community bekommen. Du kannst Fragen in unserem `#Support`-Forum stellen oder unseren speziellen `#starlight`-Channel besuchen, um aktuelle Entwicklungen und mehr zu diskutieren.
+Du kannst auch nach bekannten [Starlight-Problemen auf GitHub] (https://github.com/withastro/starlight/issues) suchen und im [Astro Discord] (https://astro.build/chat/) Hilfe von unserer aktiven, freundlichen Community erhalten! Du kannst Fragen in unserem `#Support`-Forum stellen oder unseren speziellen `#starlight`-Channel besuchen, um aktuelle Entwicklungen und mehr zu diskutieren.
```

Comment:
Vielen Dank @trueberryless! Hier brauchen wir eine kleine Verbesserung damit die Links funktionieren:

```suggestion
Du kannst auch nach bekannten [Starlight-Problemen auf GitHub](https://github.com/withastro/starlight/issues) suchen und im [Astro Discord](https://astro.build/chat/) Hilfe von unserer aktiven, freundlichen Community erhalten! Du kannst Fragen in unserem `#Support`-Forum stellen oder unseren speziellen `#starlight`-Channel besuchen, um aktuelle Entwicklungen und mehr zu diskutieren.
```

Aktuell sieht es so aus:

<img width="756" alt="image" src="https://github.com/withastro/starlight/assets/357379/4e6c7881-7652-46d6-b608-dab0255a6282">

Reply:
Okay, should I create another pull request or what should I do?

Reply:
Oh, the new commits will be automatically in this PR, I didn't know that... Nice

Reply:
Yes! New commits to the same branch will be added to the PR when you push them. (You could also have used GitHub’s UI and the <kbd>Commit suggestion</kbd> button below my comment.)

We have quite a bit of extra information about PRs and GitHub forks in our docs contributor guide in case that’s helpful! https://github.com/withastro/docs/blob/main/CONTRIBUTING.md#making-prs-pull-requests